test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -445,7 +445,8 @@ class cursor_bound_01 : public test {
     validate_search_near_not_found(
       scoped_cursor &normal_cursor, const bound &lower_bound, const bound &upper_bound)
     {
-        int ret, exact;
+        int ret = 0, exact = 0;
```

Comment:
np: I think only `exact` needed to be set but it's always good to init stuff anyway

Reply:
yeah I feel like we should just be extra cautious because of coverity 😆